src/SFA.DAS.EmployerAccounts/Commands/AddPayeToAccount/AddPayeToAccountCommandValidator.cs

```diff
@@ -47,10 +47,6 @@ public async Task<ValidationResult> ValidateAsync(AddPayeToAccountCommand item)
 
         private static void CheckFieldsArePopulated(AddPayeToAccountCommand item, ValidationResult validationResult)
         {
-            if (string.IsNullOrEmpty(item.AccessToken))
```

Comment:
What was the need to remove this validation? Is this down to the fact sometimes we are arriving via hashed account id and so the access token isn't present? but we will now always have the hashed account id regardless?